apps/meteor/client/views/room/MessageList/MessageList.tsx

```diff
@@ -25,7 +25,8 @@ type MessageListProps = {
 
 export const MessageList = ({ rid }: MessageListProps): ReactElement => {
 	const t = useTranslation();
-	const messages = useMessages({ rid });
+	const userSubscription = useUserSubscription(rid);
+	const messages = useMessages({ rid, subscription: userSubscription });
 	const subscription = useRoomSubscription();
```

Comment:
What is the difference between `userSubscription` and `subscription` here? I think they should be the same thing.

Reply:
The Room subscription does not has the translation preference for EACH user, that why I had to use the user subscription